server/telemetry.go

```diff
@@ -0,0 +1,51 @@
+package main
```

Comment:
Adding telemetry is not related to this task. Also, I am not sure how we can test it. Lets remove it from this PR

Reply:
Sure